policy_check.go

```diff
@@ -101,6 +101,7 @@ type PolicyResult struct {
 	Result         bool `jsonapi:"attr,result"`
 	SoftFailed     int  `jsonapi:"attr,soft-failed"`
 	TotalFailed    int  `jsonapi:"attr,total-failed"`
+	Sentinel       any  `jsonapi:"attr,sentinel"`
```

Comment:
it’s just interface{} afaik. can be type asserted into anything. i think it’s appropiate since the api does not define a type for it. i’ll output and paste some reflection debug later tonight to show how this unmarshalls from the tests